type RejectCallback = (arg0: NodeJS.ErrnoException) => void;
type ResolveCallback = (arg0: unknown) => void;

export { RejectCallback, ResolveCallback };
